### ISSUE TITLE
How can I use lightbox inside webview?

### ISSUE THREAD

**Issue by georgi-kovachev** [oblador/react-native-lightbox#48]
How can I use lightbox inside webview?

In webview I include like this

const injectScript = (html, bgColor) => "<body style='background-color: " + bgColor + ";'>" + script 
    + '<Lightbox underlayColor="white" navigator={this.props.navigator}>'
        +  '<Image style="height: 200px; width: 200px" '
        +  'resizeMode="contain" '
        +  'src="https://www.yayomg.com/wp-content/uploads/2014/04/yayomg-pig-wearing-party-hat.jpg" '
        +  '/> '
        +   '</Lightbox>' + "</body>";

render() {
        const {html, style, bgColor, ...otherProps} = this.props;

        // if (!BODY_TAG_PATTERN.test(html)) {
        //     throw new Error("Cannot find </body> from: " + html);
        // }
        // 
        // console.log(bgColor);

        return (
            <WebView
                {...otherProps}
                source={{html: injectScript(html, bgColor)}}
                scrollEnabled={false}
                style={[style, {height: this.state.documentHeight}]}
                onNavigationStateChange={this.handleNavigationChange}
            />
        );
    }

But it doesn't work

**Comment by adielhercules** [comment]
No, you cant. You can only pass HTML valid DOM elements to a WebView such as: div, img, table, and so on.
